src/hdusd/export/world/__init__.py

```diff
@@ -0,0 +1,143 @@
+#**********************************************************************
+# Copyright 2020 Advanced Micro Devices, Inc
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#********************************************************************
+from dataclasses import dataclass
+import numpy as np
+
+import bpy
+import mathutils
+
+from pxr import Gf, Sdf, UsdGeom, UsdLux, Tf
+
+from ...utils.image import cache_image_file
+from ...utils import BLENDER_DATA_DIR
+
+from ...utils import logging
+log = logging.Log(tag='export.world')
+
+
+@dataclass(init=False, eq=True, repr=True)
+class WorldData:
+    """ Comparable dataclass which holds all environment settings """
+
+    color: tuple = (0.0, 0.0, 0.0)
+    image: str = None
+    intensity: float = 1.0
+    rotation: tuple = (0.0, 0.0, 0.0)
+
+    @staticmethod
+    def init_from_world(world: bpy.types.World):
+        """ Returns WorldData from bpy.types.World """
+        data = WorldData()
+
+        if not world.use_nodes:
+            data.color = tuple(world.color)
+            return data
+
+        output_node = next((node for node in world.node_tree.nodes
+                           if node.bl_idname == 'ShaderNodeOutputWorld' and node.is_active_output),
+                           None)
+        if not output_node:
+            return data
+
+        from .nodes import ShaderNodeOutputWorld
+
+        node_parser = ShaderNodeOutputWorld(world, output_node)
+        node_item = node_parser.export()
+        if not node_item:
+            return data
+
+        d = node_item.data
```

Comment:
I would prefer something more understandable, e.g. "color_data", because it's used in several lines.